tutorials/tutorial-howto-monitor-django-prometheus.md

```diff
@@ -0,0 +1,230 @@
+# Tutorial: How to monitor a Django application with Prometheus
+
+>:TOPLIST:
+> - [Introduction](#introduction)
+> - [Prerequisites](#prereqs)
+> - [Step 0 - Set up a basic Django application (optional)](#step0)
+> - [Step 1 - Export prometheus-style monitoring metrics from your
+Django application](#step1)
+> - [Step 2 - Point Prometheus to your Django application metrics endpoint](#step2)
+> - [Step 3 - Instrument additional aspects of your application (optional)](#step3)
+> - [Next Steps](#next-steps)
+
+### Introduction [](introduction)
+Prometheus is an open-source systems monitoring and alerting toolkit that can be
+used to easily and cheaply monitor infrastructure and applications. In this
+tutorial we show how to monitor a Django application with Prometheus. (And even
+if you don’t have a Django application, we include an optional step to create
+one so that everyone can follow along.)
+
+### Prerequisites [](prereqs)
+A machine with the following installed:
+
+- Python
+- [pip][get-pip]
+- A locally running [Prometheus][get-prometheus] instance
+
+>:Tip: Note: Since machines commonly have multiple versions of Python installed, 
+in this tutorial we will call pip using the `python -m pip [foo]` syntax instead 
+of the `pip [foo]` syntax. This is to ensure that pip installs new components for
+the version of Python that we are using.
+
+### Step 0 - Set up a basic Django application (optional) [](step0)
+*(Please skip this step if you already have a Django application.)*
+
+#### Install [Django][get-django]
+```bash
+python -m pip install Django
+```
+
+#### Create a template project
+```bash
+cd [to the directory where you want to create the project]
```

Comment:
```suggestion
cd {your-project-directory}
```

Reply:
Is this command really necessary?  You already state that the second command creates a directory in the current directory

Reply:
You could simply say 'navigate to the directory where you want to create the project' before the code block, or just leave it out.